### ISSUE TITLE
Hebrew char taf ת (accessible by comma key on hebrew keyboard) appears in settings

### ISSUE THREAD

**Issue by schorrm** [microsoft/terminal#3733]
# A quick note:

First off, loving v. 0.7. Secondly, this is a ridiculously minor bug that I haven't been able to even reproduce on my own machine. Don't really bother fixing it, I guess. But it's still undefined behavior, and I thought you might want to know that something like this can happen.

# Environment

```none
Windows: Microsoft Windows [Version 10.0.18362.476]
Windows Terminal version (if applicable): 0.7.3291.0

Any other software?
```

# Steps to reproduce

Not sure, haven't been able to reproduce it myself. Definitely 

# Expected behavior

Should be Ctrl+comma displayed, IIRC.

# Actual behavior

A quick note: the character here, the hebrew letter taf - ת, is comma on the hebrew keyboard

![image](https://user-images.githubusercontent.com/16987694/69706278-0fbdf180-1100-11ea-87b7-e15cffb6e8e1.png)

**Comment by zadjii-msft** [comment]
Thanks for the report! That's certainly curious - probably an artifact of us looking up the string from the OS for the key that's been bound. It must be the case that the `,` doesn't roundtrip on that keyboard. @schorrm are you using an actual physical Hebrew keyboard, or another keyboard with the Hebrew keyboard layout installed?

I'm marking this Help-Wanted if someone's passionate about this bug and wants to give us a hand with it ☺️

**Comment by schorrm** [comment]
Just shift+alt on normal laptop keyboard

**Comment by DHowett-MSFT** [comment]
Yeah, this is caused by us asking Windows to map the `VK` back to the character on the keyboard that is set as the user's default keyboard. It's not great... but maybe we need to consider how to deal with keybindings in a way that lets us separate VKs from characters?

**Comment by schorrm** [comment]
Hebrew is not my default keyboard, which explains why I haven't been able to reproduce

**Comment by dkter** [comment]
I'm a fan of how Terminal handles shortcuts like this, btw:

![image](https://user-images.githubusercontent.com/34610081/84579998-ecf3e180-ada0-11ea-813a-27604472f6fd.png)

I have the numbers and shift-numbers switched on my keyboard (with MSKLC) and Windows Terminal was one of the only programs that picked up on this and switched the keyboard shortcuts without me having to change anything.

Strange how it displays like that when Hebrew isn't the default keyboard though.
